File: Remove duplicate element from sorted Linked List[28-August-2023]/Solution.java
// >> JAVA CODE <<

class GfG
{
    //Function to remove duplicates from sorted linked list.
    Node removeDuplicates(Node head)
    {
	    Node ans=new Node(-1);
	    Node ansptr=ans;
	    while(head!=null)
	    {
	        if(head.data!=ansptr.data)
	        {
	            ansptr.next=head;
	            ansptr=ansptr.next;
	        }
	        
	        head=head.next;
	    }
	    
	    ansptr.next=null;
	    
	        return ans.next;
    }
}

File: Remove duplicate element from sorted Linked List[28-August-2023]/solution.py
#  >> PYTHON CODE <<

def removeDuplicates(head):
    ans=Node(-1)
    ansptr=ans
    while(head!=None):
        if(head.data!=ansptr.data):
            ansptr.next=head
            ansptr=ansptr.next
        head=head.next
    
    ansptr.next=None
    
    return ans.next
